syntax = "proto3";

service Berkeley {
    rpc Connect(ConnectionRequest) returns (ConnectionResponse) {}
    rpc SyncClocks(TimeRequest) returns (TimeResponse) {}
}

message ConnectionRequest {
    string client_id = 1;
    double time = 2;
}
message ConnectionResponse {
    string message = 1;
}


message TimeRequest {
    string client_id = 1;
    double time = 2;
}

message TimeResponse {
    string message = 1;
    double time = 2;
}